sonoff.server.module.js

```diff
@@ -1,5 +1,5 @@
 module.exports.createServer = function (config) {
-    const CONNECTION_IS_ALIVE_CHECK_INTERVAL = 30000;
+    const CONNECTION_IS_ALIVE_CHECK_INTERVAL = 5000;
```

Comment:
i know you want to see immediately if a device is offline. But this will send a message every 5 secounds to all devices. This is a little bit too much. If you want this short timings, it would be best to add it to a config and let everybody choose how long they want the interval